src/Symfony/Component/String/AbstractUnicodeString.php

```diff
@@ -214,6 +214,14 @@ public function join(array $strings, string $lastGlue = null): static
         return $str;
     }
 
+    public function kebab(): static
+    {
+        $str = $this->camel();
```

Comment:
camel()->lower()->replace('_', '-')

Is not enough? Just asking

Reply:
If that's the case, maybe we don't need yet another method?

Reply:
The snake() function uses the same process:
https://github.com/symfony/symfony/blob/6.3/src/Symfony/Component/String/AbstractUnicodeString.php#L349

Reply:
> camel()->lower()->replace('_', '-')
> 
> Is not enough? Just asking

Your example does not work.

Reply:
Right, it's `snake()->replace('_', '-')`, I think.